moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -432,17 +428,20 @@ void MotionPlanningDisplay::changedMetricsTextHeight()
 void MotionPlanningDisplay::displayTable(const std::map<std::string, double>& values, const Ogre::ColourValue& color,
                                          const Ogre::Vector3& pos, const Ogre::Quaternion& orient)
 {
-  // the line we want to render
-  std::stringstream ss;
-  for (const std::pair<const std::string, double>& value : values)
-    ss << boost::format("%-10s %-4.2f") % value.first % value.second << '\n';
-
-  if (ss.str().empty())
+  if (values.size() == 0)
```

Comment:
```suggestion
  if (values.empty())
```

Reply:
That's also what clang-tidy tells you to do, so it is not optional :sweat_smile: 

```bash
clang_tidy_check_moveit_ros_visualization
  Filtering for files that actually changed since main
  run clang-tidy for 2/27 file(s) in 2 process(es).
  /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp:431:7: warning: the 'empty' method should be used to check for emptiness instead of 'size' [readability-container-size-empty]
    if (values.size() == 0)
        ^~~~~~~~~~~~~~~~~~
        values.empty()
  /home/runner/work/moveit2/moveit2/.work/target_ws/src/moveit2/moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp:431:7: note: FIX-IT applied suggested code changes
  /usr/lib/gcc/x86_64-linux-gnu/11/../../../../include/c++/11/bits/stl_map.h:465:7: note: method 'map'::empty() defined here
        empty() const _GLIBCXX_NOEXCEPT
        ^
  clang-tidy applied 1 of 1 suggested fixes.
  Suppressed 10519 warnings (10517 in non-user code, 2 NOLINT).
  Suppressed 10331 warnings (10331 in non-user code).
```

Reply:
The suggestion is committed.